scripts/check_openfire.py

```
@@ -2,13 +2,13 @@
 
 import sys
 
-import httpx
+import requests
 
 
 def main() -> int:
     """Go Main Go"""
     try:
-        resp = httpx.get("http://xmpp.weather.im:7070")
+        resp = requests.get("http://xmpp.weather.im:7070")
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)